ibm_db2/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+***Added***:
+
+* Add the ability to pass additional connection parameters ([#14110](https://github.com/DataDog/integrations-core/pull/14110))
```

Comment:
Missed the opportunity to mend this while I was rebasing, since we reduced the scope of the PR to just `connection_timeout` the changelog entry should reflect that.

```suggestion
* Add option to control the connection timeout ([#14110](https://github.com/DataDog/integrations-core/pull/14110))
```